File: app/src/main/java/com/lostincontext/data/GridBottomSheetItem.java
package com.lostincontext.data;


import android.support.annotation.DrawableRes;

import com.lostincontext.ruledetails.RuleDetailsPresenter;

public class GridBottomSheetItem {

    public final String name;
    @DrawableRes public final int drawableRes;
    public final RuleDetailsPresenter.Picker picker;

    public GridBottomSheetItem(String name,
                               @DrawableRes int drawableRes,
                               RuleDetailsPresenter.Picker picker) {
        this.name = name;
        this.drawableRes = drawableRes;
        this.picker = picker;
    }
}

File: app/src/main/java/com/lostincontext/data/GridBottomSheetItem.kt
package com.lostincontext.data


import android.support.annotation.DrawableRes

import com.lostincontext.ruledetails.RuleDetailsPresenter

data class GridBottomSheetItem(val name: String,
                               @DrawableRes val drawableRes: Int,
                               val picker: RuleDetailsPresenter.Picker)
